content/docs/started/k8s/kfctl-existing-arrikto.md

```diff
@@ -1,22 +1,156 @@
 +++
-title = "Kubeflow Deployment with kfctl_existing_arrikto"
-description = "Instructions for installing Kubeflow with kfctl_existing_arrikto config"
+title = "Multi-user, auth-enabled Kubeflow with kfctl_existing_arrikto"
+description = "Instructions for installing Kubeflow with kfctl_existing_arrikto.yaml config"
 weight = 4
 +++
 
-This deployment uses [Dex](https://github.com/dexidp/dex) and [Istio](https://istio.io/) for vendor-neutral authentication.
+Follow these instructions if you want to install Kubeflow on an existing Kubernetes cluster.
 
-**Maintainer and supporter: Arrikto**
+This installation of Kubeflow is maintained by
+[Arrikto](https://www.arrikto.com/), it is geared towards existing Kubernetes
+clusters and does not depend on any cloud-specific feature.
+
+In this reference architecture, we use [Dex](https://github.com/dexidp/dex) and
+[Istio](https://istio.io/) for vendor-neutral authentication.
+
+This deployment works well for on-prem installations, where companies/organizations need LDAP/AD integration for multi-user authentication, and they don't want to depend on any cloud-specific feature.
+
+![kfctl_existing_arrikto_architecture](../../kfctl_existing_arrikto-architecture.svg)
+
+Read the relevant [article](https://journal.arrikto.com/kubeflow-authentication-with-istio-dex-5eafdfac4782) for more info about this architecture.
 
-![platform existing architecture](https://i.imgur.com/OlaN73j.png)
 
 ### Prerequisites
-- Kubernetes Cluster with LoadBalancer support. Refer [Load Balancer](https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/) guide.
 
+- Kubernetes Cluster with LoadBalancer support.
+
+If you don't have support for LoadBalancer on your cluster, please
+follow the instructions below to deploy MetalLB in [Layer 2
+mode](https://metallb.universe.tf/configuration/#layer-2-configuration).
+
+<details>
+
+<summary>MetalLB deployment</summary>
+
+1. Apply the manifest:
+      
+    {{< highlight bash >}}
+    kubectl apply -f https://raw.githubusercontent.com/google/metallb/v0.8.1/manifests/metallb.yaml
+    {{< /highlight >}}
+  
+1. Allocate a pool of addresses on your local network for MetalLB to use. You
+   need at least one address for the Istio Gateway. This example assumes
+   addresses `10.0.0.100-10.0.0.110`. *You must modify these addresses based on
+   your environment*.
+
+    {{< highlight bash >}}
+    cat <<EOF | kubectl apply -f -
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      namespace: metallb-system
+      name: config
+    data:
+      config: |
+        address-pools:
+        - name: default
+          protocol: layer2
+          addresses:
+          - 10.0.0.100-10.0.0.110
+    EOF
+    {{< /highlight >}}
+
+
+#### Ensure that MetalLB works as expected (optional)
```

Comment:
This heading stands out a bit too much, especially as there isn't a matching heading for the previous section of the folded text. Also, the heading appears in the mini table of contents on the right even when the folded section is not expanded.

Suggestion: Make this sentence bold text instead of a heading:

```
**Ensure that MetalLB works as expected (optional):**
```

And also add a matching  heading at the top of the folded section:

```
**Deploy MetalLB:***
```

Reply:
thanks